openqa-investigate

```diff
@@ -27,6 +27,18 @@ jq_output_limit="${jq_output_limit:-15}"
 curl_args=(-L --user-agent "openqa-investigate")
 echoerr() { echo "$@" >&2; }
 
+case "$host" in
+    openqa.opensuse.org)
+        investigation_gid="132"
+        ;;
+    openqa.suse.de)
+        investigation_gid="637"
+        ;;
+    *)
+        investigation_gid="0"
+        ;;
```

Comment:
Wow, that's … not pretty and fragile. Can we not look up job groups by name?

Reply:
And I would make it configurable (e.g. turn the feature on/off completely).
Maybe configuring the group id here would also be easier

Reply:
Yes, you can use the default-fallback assignment but that does not help the problem that you hardcode instance specific group-ids here. Unless you find a way to use a job group name by convention I think you should go back to consider "plan A" from the ticket instead

Reply:
My suggestion was to remove the instance specific assignments and only do `investigation_gid=${investigation_gid:-0}` at the top of the script. then it can be configured in openqa.ini.
A very simple change that doesn't cost us anything.

If it turns out that we want to use it, we can always think of finding the group by name later.